src/go/k8s/kind.yaml

```diff
@@ -2,20 +2,20 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
-  image: vectorizedcloud/kindestnode:v1.21.1-pr2515
+  image: kindest/node:v1.24.7
```

Comment:
```suggestion
  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
```

i'm wondering if we need to specify the exact sha256 that is called out in the latest [kind v0.17.0](https://github.com/kubernetes-sigs/kind/releases/tag/v0.17.0) release notes.

and to also update the kind version on vtools: https://github.com/redpanda-data/vtools/pull/1342

Reply:
Yes, please do.